docs/_includes/common/flags.html

```diff
@@ -1,8 +1,29 @@
 <div id="flags">
     <h2>Flags</h2>
     <p>Skin has the ability to switch between design system versions.</p>
-    <p>DS6 is the current design system. To opt into DS4, Lasso.js requires the <span class="highlight">ds-4</span> <a href="https://github.com/lasso-js/lasso#conditional-dependencies">conditional dependency flag</a>.</p>
+    <p>DS6 is the current design system.</p>
```

Comment:
We should probably mention that you can access older design systems via `ds-x` flags.

Reply:
I'm fine with your comment Dylan, as long as we specifically call out `ds-4`, otherwise it can be ambiguous and confusing to readers. 
If we have multiple legacy design systems, then yes this makes sense, but we only have 1 at the moment, we should call it out explicitly.